packages/vscode-extension/src/utils/commonUtils.ts

```diff
@@ -211,3 +217,80 @@ export function getAllFeatureFlags(): string[] | undefined {
 export function getIsExistingUser(): string | undefined {
   return ext.context.globalState.get<string>(UserState.IsExisting);
 }
+
+export function getDefaultEnv(): string {
+  return isMultiEnvEnabled() ? "dev" : "default";
+}
```

Comment:
As we already have `environmentManager.getDefaultEnvName()`, there's no need to create a duplicated one?

Reply:
It makes sense. Let me fix it